static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -21,7 +21,7 @@ export default class FieldsOfStudySelectField extends React.Component {
   static autocompleteStyleProps = {
     menuStyle: {maxHeight: 300, width: 600},
     listStyle: {width: '100%'},
```

Comment:
Can you remove this line too, and add `menuHeight: 300`? Basically I'm looking to make it the same as `SelectField.autocompleteStyleProps`, and to fix the field I forgot about before